app/test/e2e/subscriptions-find-by-ids.spec.js

```diff
@@ -0,0 +1,99 @@
+        const noTokenResponse = await requester.post(`/api/v1/subscriptions/find-by-ids`).send();
+        noTokenResponse.status.should.equal(401);
+
+        const userResponse = await requester.post(`/api/v1/subscriptions/find-by-ids`).send({ loggedUser: ROLES.USER });
+        userResponse.status.should.equal(401);
+
+        const managerResponse = await requester.post(`/api/v1/subscriptions/find-by-ids`).send({ loggedUser: ROLES.MANAGER });
+        managerResponse.status.should.equal(401);
+
+        const adminResponse = await requester.post(`/api/v1/subscriptions/find-by-ids`).send({ loggedUser: ROLES.ADMIN });
+        adminResponse.status.should.equal(401);
+
+        const superAdminResponse = await requester.post(`/api/v1/subscriptions/find-by-ids`).send({ loggedUser: ROLES.SUPERADMIN });
+        superAdminResponse.status.should.equal(401);
+
+        const msResponse = await requester.post(`/api/v1/subscriptions/find-by-ids`).send({ loggedUser: ROLES.MICROSERVICE });
+        msResponse.status.should.equal(200);
+    });
+
+    it('Finding subscriptions providing wrong type data (non-string ids) returns a 200 OK response with no data', async () => {
+        const response = await requester
+            .post(`/api/v1/subscriptions/find-by-ids`)
+            .send({ ids: [{}], loggedUser: ROLES.MICROSERVICE });
+        response.status.should.equal(200);
+        response.body.should.have.property('data').with.lengthOf(0);
+    });
+
+    it('Finding subscriptions by ids without providing ids should return a 200 OK response with no data', async () => {
+        const response = await requester.post(`/api/v1/subscriptions/find-by-ids`).send({ loggedUser: ROLES.MICROSERVICE });
+        response.status.should.equal(200);
+        response.body.should.have.property('data').with.lengthOf(0);
+    });
+
+    it('Finding subscriptions by ids providing invalid ids should return a 200 OK response with no data', async () => {
+        const response = await requester
+            .post(`/api/v1/subscriptions/find-by-ids`)
+            .send({ ids: ['non-existing-id'], loggedUser: ROLES.MICROSERVICE });
+        response.status.should.equal(200);
+        response.body.should.have.property('data').with.lengthOf(0);
+    });
+
+    it('Finding subscriptions by ids providing existing ids should return a 200 OK response with the subscription data', async () => {
+        const subscriptionOne = await new Subscription(createSubscription(ROLES.USER.id)).save();
+        const subscriptionTwo = await new Subscription(createSubscription(ROLES.USER.id)).save();
+
+        const response = await requester
+            .post(`/api/v1/subscriptions/find-by-ids`)
+            .send({ ids: [subscriptionOne.id, subscriptionTwo.id], loggedUser: ROLES.MICROSERVICE });
+        response.status.should.equal(200);
+        response.body.should.have.property('data').with.lengthOf(2);
```

Comment:
add:

`        response.body.data.map((e) => e._id).should.have.members([subscriptionOne._id.toString(), subscriptionTwo._id.toString()]);
`